docs/supported-languages.md

```diff
@@ -103,8 +106,6 @@ Community supported languages have varying levels of rule coverage - view the re
  <summary>Click to view <strong>Semgrep OSS</strong> languages.</summary>
 
 - Bash
-- C
```

Comment:
@milanwilliams Just to confirm, are we absolutely removing all language parsing capabilities for C & C++? Put another way, are we **gating** scans for C & C++, i.e the user has to login?